backend/src/main/java/gov/cdc/usds/simplereport/service/AddressValidationService.java

```diff
@@ -0,0 +1,93 @@
+package gov.cdc.usds.simplereport.service;
+
+import static gov.cdc.usds.simplereport.api.Translators.parseState;
+import static gov.cdc.usds.simplereport.api.Translators.parseString;
+
+import com.smartystreets.api.ClientBuilder;
+import com.smartystreets.api.exceptions.SmartyException;
+import com.smartystreets.api.us_street.Candidate;
+import com.smartystreets.api.us_street.Client;
+import com.smartystreets.api.us_street.Lookup;
+import com.smartystreets.api.us_street.MatchType;
+import gov.cdc.usds.simplereport.api.model.errors.IllegalGraphqlArgumentException;
+import gov.cdc.usds.simplereport.db.model.auxiliary.StreetAddress;
+import gov.cdc.usds.simplereport.properties.SmartyStreetsProperties;
+import java.io.IOException;
+import java.util.ArrayList;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AddressValidationService {
+  public final String FACILITY_DISPLAY_NAME = "facility";
+  public final String PROVIDER_DISPLAY_NAME = "Ordering Provider";
```

Comment:
This capitalization seems slightly German. Assuming that it was intentional but drawing attention to it just in case it wasn't.

Reply:
Yes this is from the mock of errors
![image (2)](https://user-images.githubusercontent.com/53869143/110342379-5552b500-7ff9-11eb-9e50-1a950e7c352c.png)